packages/shared/src/utils/normalizePath.ts

```diff
@@ -0,0 +1,13 @@
+export const normalizePath = (path: string): string => {
+  const convertedMdPath = path.endsWith('README.md')
+    ? path.substring(0, path.length - 9)
+    : path.endsWith('.md')
+      ? path.substring(0, path.length - 3) + '.html'
+      : path
```

Comment:
BTW, if we drop markdown link support, we can have higher performance here.

I would upvote to drop support for markdown links, since we already convert every markdown link in markdown files to `<RouterLink>` (or `<VPLink>` with VPLink pr).

At lease we can add an option here?

P.S: What do you think if we add a `router` field in config files with options like :


```
router: {
  cleanUrl: true,
  resolveMarkdownLink: true,
  // ...
}
```